### DISCUSSION TITLE
Spanish Translation to docs

### DISCUSSION THREAD

**Discussion by raulwwq0** [vuejs/pinia#1886] [Ideas]
Hi! I would like to translate Pinia docs to Spanish. However, I don't know how should I do it. So if someone can tell me any suggestion about how I have to do or which steps I have to follow I will be glad to contribute with this translation.

**Comment by BenShelton** [comment]
That would be awesome! 😄 

In the PR for adding the Chinese docs there was this comment: https://github.com/vuejs/pinia/pull/1193#issuecomment-1282739152

> Thanks for your interest! Other languages will be kept in other repositories, following the guide at https://github.com/vuejs-translations/guidelines/blob/main/README.md

Although I'm not entirely sure on how that is meant to work with the Pinia docs specifically or whether it's been set up yet 🤔

**Reply by raulwwq0** [reply]
I already read it and I supposed it has to be similar, but who knows... I will wait till someone confirms it has to be done in the same way as vue translations or not. Thanks :D

**Reply by posva** [reply]
What @BenShelton said! There are of course minor adjustments to make like the repository to fork and the final URLs

**Reply by raulwwq0** [reply]
Alright then. I will start the translation as soon as possible in another repo. When it finished I will make a pr just with a link in the language drop-down to the site where the Spanish docs will be. Thanks both of you for the answers <3

**Comment by raulwwq0** [comment]
To anyone who wants to contribute to this translation here is the [repo](https://github.com/raulwwq0/pinia-spanish-docs) where I will be working. Just contact me if you have any questions.

**Reply by edgonzalez24** [reply]
I would like to support the contribution, I will contact you 👍🏽
